libs/dml/src/lift.rs

```diff
@@ -621,72 +619,72 @@ fn dml_default_kind(default_value: &ast::Expression, scalar_type: Option<ScalarT
+        Either::Left(db::OperatorClass::Int2MinMaxMultiOps) => OperatorClass::Int2MinMaxMultiOps,
+        Either::Left(db::OperatorClass::Int4BloomOps) => OperatorClass::Int4BloomOps,
+        Either::Left(db::OperatorClass::Int4MinMaxOps) => OperatorClass::Int4MinMaxOps,
+        Either::Left(db::OperatorClass::Int4MinMaxMultiOps) => OperatorClass::Int4MinMaxMultiOps,
+        Either::Left(db::OperatorClass::Int8BloomOps) => OperatorClass::Int8BloomOps,
+        Either::Left(db::OperatorClass::Int8MinMaxOps) => OperatorClass::Int8MinMaxOps,
+        Either::Left(db::OperatorClass::Int8MinMaxMultiOps) => OperatorClass::Int8MinMaxMultiOps,
+        Either::Left(db::OperatorClass::NumericBloomOps) => OperatorClass::NumericBloomOps,
+        Either::Left(db::OperatorClass::NumericMinMaxOps) => OperatorClass::NumericMinMaxOps,
+        Either::Left(db::OperatorClass::NumericMinMaxMultiOps) => OperatorClass::NumericMinMaxMultiOps,
+        Either::Left(db::OperatorClass::OidBloomOps) => OperatorClass::OidBloomOps,
+        Either::Left(db::OperatorClass::OidMinMaxOps) => OperatorClass::OidMinMaxOps,
+        Either::Left(db::OperatorClass::OidMinMaxMultiOps) => OperatorClass::OidMinMaxMultiOps,
+        Either::Left(db::OperatorClass::TextBloomOps) => OperatorClass::TextBloomOps,
+        Either::Left(db::OperatorClass::TextMinMaxOps) => OperatorClass::TextMinMaxOps,
+        Either::Left(db::OperatorClass::TimestampBloomOps) => OperatorClass::TimestampBloomOps,
+        Either::Left(db::OperatorClass::TimestampMinMaxOps) => OperatorClass::TimestampMinMaxOps,
+        Either::Left(db::OperatorClass::TimestampMinMaxMultiOps) => OperatorClass::TimestampMinMaxMultiOps,
+        Either::Left(db::OperatorClass::TimestampTzBloomOps) => OperatorClass::TimestampTzBloomOps,
+        Either::Left(db::OperatorClass::TimestampTzMinMaxOps) => OperatorClass::TimestampTzMinMaxOps,
+        Either::Left(db::OperatorClass::TimestampTzMinMaxMultiOps) => OperatorClass::TimestampTzMinMaxMultiOps,
+        Either::Left(db::OperatorClass::TimeBloomOps) => OperatorClass::TimeBloomOps,
+        Either::Left(db::OperatorClass::TimeMinMaxOps) => OperatorClass::TimeMinMaxOps,
+        Either::Left(db::OperatorClass::TimeMinMaxMultiOps) => OperatorClass::TimeMinMaxMultiOps,
+        Either::Left(db::OperatorClass::TimeTzBloomOps) => OperatorClass::TimeTzBloomOps,
+        Either::Left(db::OperatorClass::TimeTzMinMaxOps) => OperatorClass::TimeTzMinMaxOps,
+        Either::Left(db::OperatorClass::TimeTzMinMaxMultiOps) => OperatorClass::TimeTzMinMaxMultiOps,
+        Either::Left(db::OperatorClass::UuidBloomOps) => OperatorClass::UuidBloomOps,
+        Either::Left(db::OperatorClass::UuidMinMaxOps) => OperatorClass::UuidMinMaxOps,
+        Either::Left(db::OperatorClass::UuidMinMaxMultiOps) => OperatorClass::UuidMinMaxMultiOps,
+
+        Either::Right(raw) => OperatorClass::Raw(raw.to_string().into()),
```

Comment:
I actually like the `dml::` qualification here, but it's hard to justify, since we moved this module to dml a while ago, and it's only going to get smaller. Looks good :+1: